Remotion/Web/Development.WebTesting.UnitTests/Configuration/TestInfrastructureConfigurationTest.cs

```diff
@@ -26,7 +26,7 @@ namespace Remotion.Web.Development.WebTesting.UnitTests.Configuration
   [TestFixture]
   public class TestInfrastructureConfigurationTest
   {
-    private readonly string _testRequestErrorDetectionStrategyAssemblyQualifiedName = typeof (TestRequestErrorDetectionStrategy).AssemblyQualifiedName;
+    private readonly string _testRequestErrorDetectionStrategyAssemblyQualifiedName = typeof (TestRequestErrorDetectionStrategy).AssemblyQualifiedName!;
```

Comment:
Unit tests shouldn't have null checks enabled, so?
```suggestion
    private readonly string _testRequestErrorDetectionStrategyAssemblyQualifiedName = typeof (TestRequestErrorDetectionStrategy).AssemblyQualifiedName;
```